neuralprophet/forecaster.py

```diff
@@ -1687,6 +1695,18 @@ def plot(
                 figsize=figsize,
                 highlight_forecast=forecast_in_focus,
             )
+        else:  # default plotting backend
```

Comment:
if 'plotly' in plotting_backend
below: resample_active=plotting_backend == "plotly-resample"